components/BeerCounter.tsx

```diff
@@ -1,29 +1,31 @@
 import { BeerData } from "../models/types";
 
 const BeerCounter = ({ beerData }: { beerData: BeerData }) => {
-  const { count, price, maxDonation } = beerData;
-  return (
-    <div className={"flex align-center justify-center relative"}>
-      <img
-        className={"max-w-lg z-0 max-w-full"}
-        src="beeer.png"
-        alt="Picture of a beer"
-        width={700}
-      />
-      <div
-        className="text-center text-3xl md:text-5xl z-10 absolute text-black font-extrabold -ml-2"
-        style={{ top: "40%" }}
-      >
-        {count}stk
-      </div>
-      <div
-        className="text-center text-2xl md:text-4xl z-10 absolute text-black font-extrabold -ml-2"
-        style={{ top: "50%" }}
-      >
-        {Math.min(count * price, maxDonation)}kr
-      </div>
-    </div>
-  );
+	const { count, price, maxDonation } = beerData;
+	return (
+		<div className={"flex align-center justify-center relative animate-[bounce_3s_ease-in-out_infinite] "}>
+			<img
+				className={
+					"max-w-lg z-0 max-w-full "
+				}
+				src="beer.png"
+				alt="Picture of a beer"
+				width={700}
+			/>
+			<div
+				className="text-center text-3xl md:text-5xl z-10 bg-gray-200 absolute text-black font-extrabold -ml-2 p-1 rounded"
+				style={{ top: "45%" }}
```

Comment:
Love the changes to this image, I think adding just a little bit more padding to the textboxes would improve it just a little bit more (see image below)

```suggestion
				className="text-center text-3xl md:text-5xl z-10 bg-gray-200 absolute text-black font-extrabold -ml-2 p-3 rounded"
				style={{ top: "40%" }}
```

<img width="568" alt="Screenshot 2023-09-28 at 15 33 20" src="https://github.com/abakus-ntnu/charitystream/assets/45620425/e48a3060-e103-4a36-92e3-e0c5f39ea050">